scripts/dbSNP/format_dbSNP_alt_ID_database_property_schema.py

```diff
@@ -0,0 +1,110 @@
+'''
+Author:  Samantha Piekos
+Date:  09/30/20
+Name:  format_dbSNP_alt_ID_database_property_schema.py
+Description: Write all unique databases that have alternative IDs for genetic variant for a .vcf or .txt input file
+as properties of GeneticVariant.
+
+@file_input		input .vcf or .txt file of genetic variants
+@file_output	mcf output file recording all databases with alternative IDs for genetic variants as properties of GeneticVariant
+'''
+
+import sys
+import re
+
+def clean_alt_db_IDs(item):
+	'''
+	Replace unusual characters and fix typos in the names of databases with alternative IDs for genetic variants.
+	@item	string of database with alternative IDs for genetic variants than rsIDs
+	@return	cleaned up string of database with alternative IDs for genetic variants than rsIDs
+	'''
+	item = item.replace("\\x2c_", "_")  # replace unusual characters
+	item = item.replace("\\x2c", "_")
+	item = item.replace('_@_', '_')
+	item = item.replace("\\x59_", "_")
+	item = item.replace('_-_', '_')
+	if item.startswith('eiden'):
+		item = 'L' + item
+	if item.startswith('llumina'):
+		item = 'I' + item
+	item = item[0].lower() + item[1:]  # make sure first character of the name is lowercase
+	return(item)
+
+def collect_alt_db_IDs(db_IDs, set_alt_db_IDs):
+	'''
+	For each genetic variant instance add all databases with an alternative ID for it to a set.
+	@db_IDs			string of all databases with alternative IDs for a given genetic variant instance
+	@set_alt_db_IDs	set of all databases with alternative IDs for the genetic variant
+	@return			set of all databases with alternative IDs for genetic variants
+	'''
+	list_existing_ID_prop = ['dbvar', 'clinvar', 'pharmgkb']
+	pops = db_IDs.split(',')
+	for item in pops:
+		if item.lower() in list_existing_ID_prop or len(item) == 0:  # remove those with existing schema
```

Comment:
is item guaranteed to not have leading/trailing spaces? if not, its good practice to do a item.strip()

Reply:
Strip was done on line 46 of the script